zk/README.md

```diff
@@ -30,6 +30,59 @@ To configure this check for an Agent running on a host:
 
 2. [Restart the Agent][6].
 
+#### Enabling SSL
+
+Zookeeper 3.5 introduced the ability to use SSL authentication. The agent is able to connect to Zookeeper using SSL as well.
+
+One thing to note is that Zookeeper uses `truststores` and `keystores` JKS files while the Agent TLS/SSL configuration uses `tls_ca_cert`, `tls_cert`, and `tls_private_key` config options that accept PEM files.
+It may be confusing at first for those familiar with the JKS files to map to PEM files.
+
+It is assumed that one already has the server and client truststore and keystore files to start Zookeeper in SSL mode in the first place.
+More information on setting up Zookeeper in SSL mode first can be found [here][13].
+
+We will use following files as examples:
+- `server_truststore.jks`
+- `server_keystore.jks` 
+- `client_truststore.jks`
+- `client_keystore.jks`
+
+We also assume that both sides' `keystore`s and `truststore`s have each other's certificates, meaning that a Java Zookeeper client can already connect to a Zookeeper server.
+
+
+_We first will get `cert.pem` file from `client_truststore.jks`._
+1. Use `keytool` to export `client_truststore.jks` to a PKCS12 file called `cert.p12`:
+    ```
+    keytool -importkeystore -srckeystore client_truststore.jks -destkeystore cert.p12 -srcstoretype jks -deststoretype pkcs12
```

Comment:
Confused about the commands. From what I can read online, you need to use `-exportkeystore` to export the certificate to a cert file. And use `-importkeystore` to export the private key from a keystore.
I'm surprised all the commands are the same in here, can we double-check that?

Also when exporting the certs (both ca_cert and cert), couldn't we use the `rfc` flag of `exportkeystore` to squash those two operations into one?

Finally a keystore/truststore contains multiple entries, I assume we need to specify one using the `-alias` flag of the `exportkeystore` command.

Ref: https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html#keytool_option_exportcert

Reply:
I couldn't find `-exportkeystore` in that doc, but saw `-exportcert`, is that what you meant?

I tested `keytool -importkeystore` with `openssl pkcs12` and `keytool -exportcert -rfc` for the `ca_cert` and `cert` files, and I got the same results. However, I'll change it to use `-exportcert -rfc` instead since it is one fewer step like you said.

Reply:
Yup sorry, I meant exportcert